src/app/applications/applications-list/applications-table/applications-table.component.ts

```diff
@@ -110,10 +122,42 @@ export class ApplicationsTableComponent implements AfterViewInit, OnInit {
   }
 
   applicationHasDevices(id: number): boolean {
-    const applicationToDelete = this.data?.find(app => app.id === id);
+    const applicationToDelete = this.data?.find((app) => app.id === id);
     return applicationToDelete && applicationToDelete.iotDevices.length > 0;
   }
 
+  applicationHasSigFoxDevices(id: number): boolean {
+    const applicationToDelete = this.data?.find((app) => app.id === id);
+    const checkForSigfox = applicationToDelete.iotDevices.find((device) => {
+      return device.type === DeviceType.SIGFOX;
+    });
+    if (checkForSigfox) {
+      return true;
+    } else return false;
```

Comment:
Redundant if-check. 

I would probably rename 'checkForSigfox ' to 'sigfoxDevice' and then just  `return sigfoxDevice !== undefined`